travis/for-lint.sh

```diff
@@ -11,4 +11,6 @@
 mkdir -p app/code/Pulsestorm/Travis
 printf "<?php\nnamespace Pulsestorm\\Travis;\n\nclass Foo\n{\n}\n"  > app/code/Pulsestorm/Travis/Test.php
 
-# pestle.phar magento2:generate:crud-model Pulsestorm_Travis Thing
+## test generate:crud-model, rm InstallSchema backup file
+pestle.phar magento2:generate:crud-model Pulsestorm_Travis Thing
```

Comment:
I'm looking at this line -- am I correct that this PR is only concerned with the formatting on the following files?

```
app/code/Pulsestorm/Travis/Model/Thing.php
app/code/Pulsestorm/Travis/Model/ThingRepository.php
app/code/Pulsestorm/Travis/Model/ResourceModel/Thing.php
app/code/Pulsestorm/Travis/Model/ResourceModel/Thing/Collection.php
```

Reply:
9 files are generated, 8 if you remove the .bak file

app/code/Pulsestorm/Travis/Model/Thing.php
app/code/Pulsestorm/Travis/Model/ThingRepository.php
app/code/Pulsestorm/Travis/Model/ResourceModel/Thing.php
app/code/Pulsestorm/Travis/Model/ResourceModel/Thing/Collection.php

app/code/Pulsestorm/Travis/Api/ThingRepositoryInterface.php
app/code/Pulsestorm/Travis/Api/Data/ThingInterface.php
app/code/Pulsestorm/Travis/Setup/InstallSchema.php
app/code/Pulsestorm/Travis/Setup/InstallData.php

They all pass phpcs checks now.

In InstallData.php I disabled phpcs for the ->install() method (empty function)
I tried disabling just the specific rule ( // phpcs:disable Magento2.CodeAnalysis.EmptyBlock.DetectedFUNCTION ) it works locally but fails in Travis build.